task1_string_manipulation/helper.py

```diff
@@ -0,0 +1,52 @@
+
+def removeMultyOccurence(str):
+    temp = ""
+
+    for s in str:
+        if s.lower() not in temp.lower():
+            temp = temp + s
+
+    return temp
+
+def repeats(str, ch):
+    count = 0
+    for s in str:
+        if s.lower() == ch.lower():
+            count = count + 1
+
+    if count>1:
+        return True
+    else:
+        return False
+
+def removeFirst(str, ch):
+
+    index = str.find(ch)
```

Comment:
Avoid using index, str, ch as variable names.
Examples for good variable names are:
`index_for_first_occurence`
`first_occuring_character` etc.